go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/NYTimes/gziphandler v1.0.1
 	github.com/armon/circbuf v0.0.0-20150827004946-bbbad097214e
 	github.com/armon/go-metrics v0.3.4
+	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
```

Comment:
Hm, Consul appears to use https://pkg.go.dev/github.com/miekg/dns#IsDomainName in at least one place. Wonder if there's any reason to standardize.